@xen-orchestra/web-core/lib/components/title/VtsTitle.vue

```diff
@@ -0,0 +1,39 @@
+<!-- v2 -->
+<template>
+  <div class="vts-title">
+    <div class="typo h4-medium label">
+      <slot />
+    </div>
+    <div class="actions">
+      <slot name="actions" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineSlots<{
+  default(): any
+  actions(): any
```

Comment:
The `actions` slot is optional and can be updated to:
```suggestion
  actions?(): any
```